### PR TITLE
DogOrCat

### PR DESCRIPTION
**Pull request by GeorgeGorgi**
**Why**

To better society.

---

**What**

This pull request contains a new file called dog or cat.
---

**Verification**

This program shows an image of a cat or a dog.

*******

To Run the program, ensure your path leads to the
DogOrCat folder.

Once you are in the correct directory. type 'python catordog.py'.

*******
---

**Checklist:**

- [x] This contribution abides by all rules listed in `docs/Rules.md`

### CODE REVIEW

File: submissions/hello_world/DogOrCat/README.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+This program shows an image of a cat or a dog.
+
+*******
+
+To Run the program, ensure your path leads to the
+DogOrCat folder.
+
+Once you are in the correct directory. type 'python catordog.py'.
+
+*******
```
Comment (SoorajModi) [inline submissions/hello_world/DogOrCat/README.txt:10]: Your README should be a markdown file, and missing the following info:
- List of contributors
- Why you contributed it

**Comment by SoorajModi** [comment]
Also, improve the name of the README and the format of the pull request template, take a look at #1 for an example submission

**Review by SoorajModi — CHANGES_REQUESTED** [review]

File: submissions/hello_world/DogOrCat/DogOrCat.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+        elif x == 2:
+            webbrowser.open(imgCat)
+            break
+        else:
+            print("\n Error in selection. Select Again.\n")
+
+print("Hehehe")
+           
+   
+
```
Comment (MarshallAsch) [inline submissions/hello_world/DogOrCat/DogOrCat.py:20]: remove excess whitespace
```suggestion
```

File: submissions/hello_world/DogOrCat/README.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+This program shows an image of a cat or a dog.
+
+*******
```
Comment (MarshallAsch) [inline submissions/hello_world/DogOrCat/README.txt:3]: You can use markdown section blocks here. See [here](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax)

File: submissions/hello_world/DogOrCat/README.txt
Comment (MarshallAsch) [inline submissions/hello_world/DogOrCat/README.txt:1]: This should be be a `.md` file not a `.txt` file

**Review by MarshallAsch — CHANGES_REQUESTED** [review]

**Comment by SoorajModi** [comment]
Hey @Bonew13, there are a few outstanding requests for change in this PR, please try to have these issues resolved ASAP so we can merge in your contribution

**Pull request closed without merging**